flink-dist/src/test/java/org/apache/flink/dist/BashJavaUtilsITCase.java

```diff
@@ -72,20 +79,23 @@ public void testGetTmResourceParamsConfigsWithDynamicProperties() throws Excepti
 
 	@Test
 	public void testGetJmResourceParams() throws Exception {
-		int expectedResultLines = 1;
+		int expectedResultLines = 2;
```

Comment:
I guess that this commit should be merged together with the previous one because w/o, the previous commit should fail in this test here.

Reply:
Yup, I planned to squash all the commits in a proper way after the review is done.